spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/refresher_spec.rb

```diff
@@ -74,16 +77,23 @@ def assert_vm_labels
     expect(vm.labels.count).to eq(4)
   end
 
+  def assert_specific_cloud_volume_type
+    cvt = ems.cloud_volume_types.find_by(:ems_ref => 'general-purpose')
+
+    expect(cvt.name).to eq('general-purpose')
+    expect(cvt.description).to eq('tiered')
+  end
+
   # Test the components of a cloud subnet.
   def assert_specific_cloud_subnet
     cloud_subnet = ems.cloud_subnets.find_by(:ems_ref => '0757-ef523a2f-5356-42ff-8a78-9325509465b9')
 
     # Test cloud_network relationship.
-    cloud_network = ems.cloud_network.find_by(:ems_ref => 'r014-0fa2acc6-2a41-4f2b-9c89-bcea07cdcbc3')
```

Comment:
:eyes: how was this passing before?

Reply:
Oh it wasn't being called before, missed that in review my bad

Reply:
Bah, did that get reverted again. I fixed that and then did a rebase.